cloud/spawn.go

```diff
@@ -324,15 +324,10 @@ func ModifySpawnHost(ctx context.Context, env evergreen.Environment, host *host.
 func MakeExtendedSpawnHostExpiration(host *host.Host, extendBy time.Duration) (time.Time, error) {
 	if err := host.PastMaxExpiration(extendBy); err != nil {
 		return time.Time{}, err
+	} else {
```

Comment:
Since the if returns, I don't think putting this in an else helps.

Reply:
done